xonsh/environ.py

```diff
@@ -937,13 +937,13 @@ def _first_branch_timeout_message():
           file=sys.stderr)
 
 
-def current_branch(pad=True):
+def current_branch():
```

Comment:
I don't think that the API should be here unless @melund approves.  I believe that he, and others, may use this.  Instead, maybe change `pad=NotImplemented` and then print a deprecation warning if `pas is not NotImplemented.`